### DISCUSSION TITLE
Bass unknown error after latest windows update

### DISCUSSION THREAD

**Discussion by nzxl101** [ppy/osu#16881] [Q&A] [answered]
Hey,
I'm having trouble running lazer with audio after latest windows update.

I didn't change any OS settings or installed third party apps in the last few days and I use the default installed windows drivers.
Just checked this on stable because I was curious and stable automatically toggles the legacy audio engine, not sure if this a problem on my setup or generally on windows.

![image](https://user-images.githubusercontent.com/63413558/154191892-99712849-91c4-49ff-b6f0-cc71dfb8684a.png)
![image](https://user-images.githubusercontent.com/63413558/154192176-72fef826-4d80-4667-a0d1-d0eab8e1695f.png)

[performance.log](https://github.com/ppy/osu/files/8076518/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8076519/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8076520/updater.log)
[network.log](https://github.com/ppy/osu/files/8076521/network.log)

**Comment by peppy** [comment, marked as answer]
That's quite weird. Can you take screenshots of the other audio control panels (that show the active playback devices) like below:

![Parallels Desktop 2022-02-16 at 07 27 29](https://user-images.githubusercontent.com/191335/154216373-63d12f55-5654-4433-8ff6-c62a6c22fe96.png)

**Reply by nzxl101** [reply]
https://user-images.githubusercontent.com/63413558/154265365-784a7543-3d5a-4dde-a2c9-f47c3763e961.mp4

**Reply by nzxl101** [reply]
I think I've found the issue. It's related to the MSI Driver App, after uninstalling the MSI drivers, lazer and stable audio just work fine again.
It wasn't an issue before, so I don't really know why it started to F up. Might not be a thing that you can fix, so this can be closed.

**Reply by peppy** [reply]
Glad to hear you fixed the issue. It's definitely going to be hard to figure out what was going on here without reproducing the state locally (which I'm not sure we'll be able to do). Probably best just forget this happened unless someone else hits the same issue in the future.
